src/coreclr/jit/fgprofile.cpp

```diff
@@ -2628,6 +2671,9 @@ void flowList::setEdgeWeights(BasicBlock::weight_t theMinWeight, BasicBlock::wei
 {
     assert(theMinWeight <= theMaxWeight);
 
+    JITDUMP("Setting edge weights for BB?? -> " FMT_BB " to [" FMT_WT " .. " FMT_WT "]\n", getBlock()->bbNum,
+            theMinWeight, theMaxWeight);
+
```

Comment:
This is backwards; I'll fix in a subsequent PR (and pass in the actual dest block to remove the `BB??`), if that's ok.

Reply:
Or maybe just merge in my next round of changes here.

Reply:
Fixed with latest commit.